docs/reference/rest-api/common-parms.asciidoc

```diff
@@ -148,11 +148,16 @@ The destination for the {transform}.
 end::dest[]
 
 tag::dest-index[]
-The _destination index_ for the {transform}. The mappings of the destination
+The _destination index_ for the {transform}.
+In case of a `pivot` transform, the mappings of the destination
 index are deduced based on the source fields when possible. If alternate
 mappings are required, use the
 https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-create-index.html[Create index API]
 prior to starting the {transform}.
+In case of a `latest` transform, the mappings are never deduced. If dynamic
+mappings for the destination index are undesirable, use the
+https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-create-index.html[Create index API]
```

Comment:
It should be possible to link this using a tag rather than a full URL. It will be more robust to future changes in the structure of the website.

Reply:
Done.